package.json

```diff
@@ -90,6 +90,7 @@
     "ember-event-helpers": "^0.1.1",
     "ember-export-application-global": "2.0.1",
     "ember-fetch": "8.1.2",
+    "ember-keyboard": "^8.2.0",
```

Comment:
Not a fan of adding a whole dep for just one line `{{on "keyup"}}`  for example should also work

Reply:
My motivation was to keep it a *template only component*, but since the `this.args.onToggle` assertions would make the template also more readable, I guess it's time to create this `component.js` file. :sweat_smile: